crates/sui-bridge/src/action_executor.rs

```diff
@@ -715,15 +695,17 @@ mod tests {
             _handles,
             gas_object_ref,
             sui_address,
-            token_tx,
+            sui_token_type_tags,
             _bridge_pause_tx,
         ) = setup().await;
         let (action_certificate, _, _) = get_bridge_authority_approved_action(
             vec![&mock0, &mock1, &mock2, &mock3],
             vec![&secrets[0], &secrets[1], &secrets[2], &secrets[3]],
+            None,
+            true,
         );
         let action = action_certificate.data().clone();
-        let id_token_map = token_tx.borrow();
+        let id_token_map = (*sui_token_type_tags.load().clone()).clone();
```

Comment:
nit: perhaps `let id_token_map = &**sui_token_type_tags.load();` is easer to read?